karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
I don't have a working example of this level of functionality not using Karma (or Mocha) at the moment. Will explore it and put together as a separate PR so we can compare which one looks better.